src/typescript/11-keyof-typeof/keyof-typeof.md

```diff
@@ -67,10 +70,13 @@ const tRex = {
   weightInKilograms: 7_000,
 };
 
-let stegosaurus: typeof tRex; // {name: string; type: string; weightInKilograms: number;}
+let stegosaurus: typeof tRex; 
 
-type Dinosaur = typeof tRex; // {name: string; type: string; weightInKilograms: number;}
+type Dinosaur = typeof tRex; 
 ```
+@highlight 7, 9
+
+Thanks to the use of `typeof`, the variable `stegosaurus` and the type `Dinosaur` both now has the type: `{name: string; type: string; weightInKilograms: number;}`
 
 There are some shortcomings to this approach since its looking at a single value, and the type returned isn’t as specific as it could be. Take for example `type` and `weightInKilograms` if we were to define the `Dinosaur` type ourselves we would want to restrict and expand those properties into something like this:
```

Comment:
`its` -> `it’s`

“Take for example” is a run-on sentence and needs some punctuation to fix it.